models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+# Create your models here.
+
+
+
+class Destination(models.Model) :
+    name = models.CharField(max_length = 100)
+    img = models.ImageField(upload_to = 'pics')
+    desc = models.TextField()
+    price = models.IntegerField()
+    offer = models.BooleanField(default = False)
+        
+class News(models.Model) :
+    SPORTS='SP'
+    ENTERTAINMENT='EN'
+    MOVIES='MO'
+    NEWS_CHOICE=[   
+    (SPORTS,'Sports'),(ENTERTAINMENT,'Entertaintment'),(MOVIES,'Movies'),
+    ]
+    id = models.IntegerField(null=False)
```

Comment:
It's should always be `id = models.IntegerField(null=False, blank=False)`. Here, null is used to tell the database that this field never be empty and blank will help Django to validate the field that it should never accept this field as empty.

Similar for date and headline field also.